config/database.yml

```diff
@@ -6,7 +6,7 @@ default: &default
   password: <%= ENV['SERVICE_APPROVAL_DATABASE_PASSWORD'] %>
   wait_timeout: 5
   min_messages: warning
-  host: localhost
+  host: <%= ENV['SERVICE_APPROVAL_DATABASE_HOST'] %>
```

Comment:
`database.yml` should be replaced by a sample [database.dev.yml](https://github.com/ManageIQ/service_portal-api/blob/master/config/database.dev.yml) so the real database.yml doesn't create dirty commits.

Reply:
You can then replace the `<%= ENV['SERVICE_APPROVAL_DATABASE_HOST'] %>` environment variable in the `docker run` command with `DATABASE_URL`.

Reply:
my goal is to have a `database.yml` that does not need modification.